libraries/aws_conn.rb

```diff
@@ -28,4 +28,12 @@ def iam_resource
   def iam_client
     @iam_client ||= Aws::IAM::Client.new
   end
+
+  def sts_client
+    @sts_client ||= Aws::STS::Client.new
+  end
+
+  def account_id
+    sts_client.get_caller_identity[:account]
```

Comment:
Yup I'm good to close this. So I'm unclear of the result. If I start in account `A` then assume a role in account `B`, is the root account provided by STS from account `A` or account `B`?